.devcontainer/Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM mcr.microsoft.com/devcontainers/miniconda:0-3
+FROM python:3.10
```

Comment:
Why Python 3.10? Maybe document this choice?

Reply:
Because using Python 3.11 will cause an error from fairseq and Python 3.10 can avoid that issue. Here's the original error message:
ValueError: mutable default <class 'fairseq.dataclass.configs.CommonConfig'> for field common is not allowed: use default_factory